pages/doc/wavefront_prometheus.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Using Prometheus Queries in Wavefront
+keywords: query language
+sidebar: doc_sidebar
+published: true
+permalink: wavefront_prometheus.html
+summary: Run Prometheus from the Wavefront query line
+---
+
+Starting with release 2020.26, Wavefront includes support for Prometheus queries.
```

Comment:
this comment applies to anywhere in the doc we use the phrase "Prometheus queries".  We should consider changing "Prometheus queries" to "PromQL queries"

Reply:
Right.